### PR TITLE
`azurerm_express_route_connection` - Support `inbound_route_map_id`, `outbound_route_map_id`, `enabled_private_link_fast_path`

### PR DESCRIPTION
**Pull request by xuzhang3**
Support `inbound_route_map_id`, `outbound_route_map_id`, `enabled_private_link_fast_path`

```log
=== RUN   TestAccExpressRouteConnection_basic
--- PASS: TestAccExpressRouteConnection_basic (5665.34s)
PASS
ok      github.com/hashicorp/terraform-provider-azurerm/internal/services/network       5705.892s


PASS
ok      github.com/hashicorp/terraform-provider-azurerm/internal/services/network       6358.210s

=== RUN   TestAccExpressRouteConnection
=== RUN   TestAccExpressRouteConnection/Resource
=== RUN   TestAccExpressRouteConnection/Resource/requiresImport
--- PASS: TestAccExpressRouteConnection (5550.84s)
    --- PASS: TestAccExpressRouteConnection/Resource (5550.84s)
        --- PASS: TestAccExpressRouteConnection/Resource/requiresImport (5550.84s)
PASS
ok      github.com/hashicorp/terraform-provider-azurerm/internal/services/network       5603.748s

=== RUN   TestAccExpressRouteConnection
=== RUN   TestAccExpressRouteConnection/Resource
=== RUN   TestAccExpressRouteConnection/Resource/complete
--- PASS: TestAccExpressRouteConnection (5472.03s)
    --- PASS: TestAccExpressRouteConnection/Resource (5472.03s)
        --- PASS: TestAccExpressRouteConnection/Resource/complete (5472.03s)
PASS
ok      github.com/hashicorp/terraform-provider-azurerm/internal/services/network       5532.557s

=== RUN   TestAccExpressRouteConnection
=== RUN   TestAccExpressRouteConnection/Resource
=== RUN   TestAccExpressRouteConnection/Resource/update
--- PASS: TestAccExpressRouteConnection (6486.12s)
    --- PASS: TestAccExpressRouteConnection/Resource (6486.12s)
        --- PASS: TestAccExpressRouteConnection/Resource/update (6486.12s)
PASS
ok      github.com/hashicorp/terraform-provider-azurerm/internal/services/network       6533.837s
```

### CODE REVIEW

File: internal/services/network/express_route_connection_resource.go [hunk 1]
```diff
@@ -68,13 +68,31 @@ func resourceExpressRouteConnection() *pluginsdk.Resource {
 				Optional: true,
 			},
 
+			"enabled_express_route_gateway_bypass": {
```
Comment (stephybun) [inline internal/services/network/express_route_connection_resource.go:71]: Can we rename this to
```suggestion
			"express_route_gateway_bypass_enabled": {
```

File: website/docs/r/express_route_connection.html.markdown [hunk 2]
```diff
@@ -95,6 +95,8 @@ The following arguments are supported:
 * `authorization_key` - (Optional) The authorization key to establish the Express Route Connection.
 
 * `enable_internet_security` - (Optional) Is Internet security enabled for this Express Route Connection?
+  
+* `enabled_express_route_gateway_bypass` - (Optional) Indicates whether FastPath enabled for Virtual Wan Firewall Hub. Defaults to `false`.
```
Comment (stephybun) [inline website/docs/r/express_route_connection.html.markdown:99]: Is that what this property does? The description doesn't sound like it matches the property name
```suggestion
* `enabled_express_route_gateway_bypass` - (Optional) Specified whether Fast Path is enabled for Virtual Wan Firewall Hub. Defaults to `false`.
```

File: website/docs/r/express_route_connection.html.markdown [hunk 3]
```diff
@@ -106,6 +108,10 @@ A `routing` block supports the following:
 
 * `associated_route_table_id` - (Optional) The ID of the Virtual Hub Route Table associated with this Express Route Connection.
 
+* `inbound_route_map_id` - (Optional) The ID of the RouteMap associated with this Express Route Connection for inbound learned routes.
+
+* `outbound_route_map_id` - (Optional) The ID of the RouteMap associated with this Express Route Connection for outbound advertised routes.
+
```
Comment (stephybun) [inline website/docs/r/express_route_connection.html.markdown:114]:
```suggestion
* `inbound_route_map_id` - (Optional) The ID of the Route Map associated with this Express Route Connection for inbound routes.

* `outbound_route_map_id` - (Optional) The ID of the Route Map associated with this Express Route Connection for outbound routes.

```

**Review by stephybun — CHANGES_REQUESTED** [review]
Thanks for this @xuzhang3, left a few comments in-line but once those are resolved we can take another look and get this merged!

**Review by katbyte — APPROVED** [review]
LGTM 🦀

**Review by stephybun — COMMENTED** [review]
Thanks @xuzhang3 LGTM 🚴

**Pull request merged**